pyControl4/director.py

```diff
@@ -87,6 +221,23 @@ async def sendPostRequest(self, uri, command, params, async_variable=True):
                     await checkResponseForError(await resp.text())
                     return await resp.text()
 
+    async def getAllItemsByCategory(self, category, query_string=None):
```

Comment:
Nice find on the Control4 API! Do you have any information on how `query_string` might work?